app/proto/search.proto

```diff
@@ -133,6 +133,7 @@ message UserSearchReq {
   google.protobuf.BoolValue drinks_at_home = 25;
   google.protobuf.BoolValue parking = 26;
   google.protobuf.BoolValue camping_ok = 27;
+  google.protobuf.BoolValue profile_completed = 31;
```

Comment:
I think this should be a `bool` like `query_name_only`. The `google.protobuf.BoolValue` allows you to communicate three states: true/false/not specified. In this case I doubt anyone wants to explicitly show profiles that are not complete, so it would suffice to have an "only show complete profiles" boolean.

Reply:
but isn't that also applicable to parking, `camping ok`- nobody wants to see the not `camping ok `users. Hence i placed this under here.

Reply:
I suppose it is. I don't think anyone ever wants to find people particularly without parking in the same way I doubt anyone wants to find people with empty profiles.

I'm fully happy for it to be whichever way you prefer. If you keep it as a `BoolValue`, then I think the python implementation needs the `HasField` bit in the other comment!

Reply:
it made sense to have it as such `bool` because we want to use it at the top of filtering, let me know if it looks ok now.